lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -449,6 +451,9 @@ <h4 class="message-title">${_('Your current verification will expire soon.')}</h
                }
            });
 </script>
+<%static:require_module module_name="js/course_sharing/course_sharing_events" class_name="CourseSharingEvents">
+    CourseSharingEvents("${course_overview.id | n, js_escaped_string}");
+</%static:require_module>
```

Comment:
Let's re define this as:
```
<%block name="js_extra">
<%static:require_module module_name="js/course_sharing/course_sharing_events" class_name="CourseSharingEvents">
    CourseSharingEvents("${course_overview.id | n, js_escaped_string}");
</%static:require_module>
</%block>
```

For more details check `common/djangoapps/pipeline_mako/templates/static_content.html` file for  require_module and require_module_async.